prisma/seed.ts

```diff
@@ -25,20 +32,55 @@ main()
 function getUsers() {
   return [
     {
+      id: '82bf2c53-a9eb-4029-8d6d-17f514e8be7b',
```

Comment:
Prisma should automatically generate `id` values automatically, so we shouldn't supply them manually in our seed data.
```suggestion
```
```suggestion
      id: '82bf2c53-a9eb-4029-8d6d-17f514e8be7b',
```